dma4500mSup/dma4500m.proto

```diff
@@ -5,8 +5,8 @@
 # European Spallation Source ERIC
 # March 2018
 
-replytimeout = 2000;
-readtimeout  = 2000;
+replytimeout = 5000;
+readtimeout  = 5000;
```

Comment:
I changed them because I was randomly getting some strange errors when running commands, such as PVs trying to parse the output of previous commands. Not sure if this would also happen during normal operation though